<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>insertLinkDialog</class>
    <widget class="QDialog" m_name="insertLinkDialog">
        <property m_name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>104</height>
            </rect>
        </property>
        <property m_name="font">
            <font>
                <pointsize>10</pointsize>
            </font>
        </property>
        <property m_name="windowTitle">
            <string>Insert Link</string>
        </property>
        <property m_name="windowIcon">
            <iconset resource="../resources/images.qrc">
                <normaloff>:/icons/books_icon.png</normaloff>:/icons/books_icon.png
            </iconset>
        </property>
        <layout class="QHBoxLayout" m_name="horizontalLayout">
            <item>
                <layout class="QGridLayout" m_name="gridLayout">
                    <item row="1" column="0">
                        <widget class="QLabel" m_name="labelLink">
                            <property m_name="text">
                                <string>Link</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <layout class="QHBoxLayout" m_name="horLayButtons">
                            <item>
                                <spacer m_name="horizontalSpacer">
                                    <property m_name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property m_name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" m_name="buttonOk">
                                    <property m_name="minimumSize">
                                        <size>
                                            <width>70</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                    <property m_name="maximumSize">
                                        <size>
                                            <width>70</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property m_name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property m_name="text">
                                        <string>Ok</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" m_name="buttonCancel">
                                    <property m_name="minimumSize">
                                        <size>
                                            <width>70</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                    <property m_name="maximumSize">
                                        <size>
                                            <width>70</width>
                                            <height>16777215</height>
                                        </size>
                                    </property>
                                    <property m_name="cursor">
                                        <cursorShape>PointingHandCursor</cursorShape>
                                    </property>
                                    <property m_name="text">
                                        <string>Cancel</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="0" column="0">
                        <widget class="QLabel" m_name="labelTitle">
                            <property m_name="text">
                                <string>Title</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" m_name="textTitle"/>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" m_name="textLink"/>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../resources/images.qrc"/>
    </resources>
    <connections/>
</ui>
